### PR TITLE
BETA release post.

### PR DESCRIPTION
**Pull request by thruflo**
https://deploy-preview-2108--electric-next.netlify.app/blog/2024/12/09/electric-beta-release

### CODE REVIEW

**Review by KyleAMathews — CHANGES_REQUESTED** [review]
Love it! Very simple and strong intro.

I made some small copy suggestions but nothing major.

File: website/blog/posts/2024-12-09-electric-beta-release.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
+---
+title: "Electric BETA release"
+description: >-
+  The Electric sync engine is now in BETA.
+  If you haven't checked out Electric recently,
+  it's a great time to take another look.
+excerpt: >-
+  With version 1.0.0-beta, the Electric sync engine is now in BETA!
```
Comment (KyleAMathews) [inline website/blog/posts/2024-12-09-electric-beta-release.md:8]:
```suggestion
  With version 1.0.0-beta.0, the Electric sync engine is now in BETA!
```

File: website/blog/posts/2024-12-09-electric-beta-release.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+        const icon = document.createElement('span')
+        icon.classList.add('vpi-electric-icon')
+
+        link.prepend(icon)
+      })
+    }
+  })
+</script>
+
+With version [`1.0.0-beta`](https://github.com/electric-sql/electric/blob/main/packages/sync-service/CHANGELOG.md) the Electric sync engine is now in BETA!
```
Comment (KyleAMathews) [inline website/blog/posts/2024-12-09-electric-beta-release.md:52]:
```suggestion
With version [`1.0.0-beta.0`](https://github.com/electric-sql/electric/blob/main/packages/sync-service/CHANGELOG.md) the Electric sync engine is now in BETA!
```

File: website/blog/posts/2024-12-09-electric-beta-release.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+</figure>
+
+You can also see how large-scale apps built with Electric feel to use with our updated [ Linearlite](/demos/linearlite) demo. This is a [Linear](https://linear.app) clone that loads 100,000k issues and their comments through Electric into PGlite. It loads fast, it feels instant and it's fully interactive:
+
+<figure>
+  <a href="https://linearlite.examples.electric-sql.com" target="_blank">
+    <img :src="LinearLiteScreenshot" />
+  </a>
+  <figcaption>
+    Screenshot of Linearlite. Click on it to
```
Comment (KyleAMathews) [inline website/blog/posts/2024-12-09-electric-beta-release.md:109]:
```suggestion
    Screenshot of Linearlite.
```

File: website/blog/posts/2024-12-09-electric-beta-release.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+  <div class="action">
+    <VPButton
+        href="/docs/intro"
+        text="Documentation"
+        theme="alt"
+    />
+  </div>
+</div>
+
+### Signup for cloud
```
Comment (KyleAMathews) [inline website/blog/posts/2024-12-09-electric-beta-release.md:209]:
```suggestion
### Signup for early access to Electric Cloud
```

File: website/blog/posts/2024-12-09-electric-beta-release.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+
+We're also working hard building our [Cloud product](/product/cloud), which provides managed Electric hosting so you don't need to [host Electric yourself](/docs/guides/deployment).
```
Comment (KyleAMathews) [inline website/blog/posts/2024-12-09-electric-beta-release.md:211]:
```suggestion
We're also building a [Electric Cloud](/product/cloud), which provides managed Electric hosting.
```

File: website/blog/posts/2024-12-09-electric-beta-release.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+
+If you're interested in using Electric Cloud, you can sign up for early access now:
```
Comment (KyleAMathews) [inline website/blog/posts/2024-12-09-electric-beta-release.md:213]:
```suggestion
We'll be opening access soon in a private beta so if interested, you can sign up for early access now:
```

**Review by KyleAMathews — COMMENTED** [review]
Some more tweaks

**Review by KyleAMathews — APPROVED** [review]
:shipit:

**Comment by thruflo** [comment]
⚠️ 🔌

File: website/docs/reference/benchmarks.md [hunk 7]
```diff
@@ -142,6 +142,8 @@ Electric is designed to run behind a CDN, using the CDN's request collapsing cap
   <ScalabilityChart />
 </figure>
 
+These statistics were generated using our [client load benchmarking](https://github.com/electric-sql/client-load-benchmarking) suite that allows for measuring client latencies for any combination of concurrent connected clients and database workload.
```
Comment (thruflo) [inline website/docs/reference/benchmarks.md:145]: Maybe `client latencies and sync service resource use`?

**Pull request merged**